src/org/zaproxy/zap/extension/fuzz/httpfuzzer/processors/FuzzerHttpMessageScriptProcessorAdapterUIHandler.java

```diff
@@ -36,226 +44,318 @@
-        }
-
-        @Override
-        public void clear() {
-            scriptComboBox.setSelectedIndex(-1);
-        }
-
-        @Override
-        public boolean validate() {
-            if (scriptComboBox.getSelectedIndex() == -1) {
-                JOptionPane.showMessageDialog(
-                        null,
-                        Constant.messages.getString("fuzz.httpfuzzer.processor.scriptProcessor.panel.warnNoScript.message"),
-                        Constant.messages.getString("fuzz.httpfuzzer.processor.scriptProcessor.panel.warnNoScript.title"),
-                        JOptionPane.INFORMATION_MESSAGE);
-                scriptComboBox.requestFocusInWindow();
-                return false;
-            }
-            return true;
-        }
-
-        private static class ScriptUIEntry implements Comparable<ScriptUIEntry> {
-
-            private final ScriptWrapper scriptWrapper;
-            private final String scriptName;
-
-            public ScriptUIEntry(ScriptWrapper scriptWrapper) {
-                this.scriptWrapper = scriptWrapper;
-                this.scriptName = scriptWrapper.getName();
-                if (scriptName == null) {
-                    throw new IllegalArgumentException("Script must have a name.");
-                }
-            }
-
-            public ScriptWrapper getScriptWrapper() {
-                return scriptWrapper;
-            }
-
-            @Override
-            public String toString() {
-                return scriptName;
-            }
-
-            @Override
-            public int hashCode() {
-                final int prime = 31;
-                int result = 1;
-                result = prime * result + ((scriptName == null) ? 0 : scriptName.hashCode());
-                return result;
-            }
-
-            @Override
-            public boolean equals(Object obj) {
-                if (this == obj) {
-                    return true;
-                }
-                if (obj == null) {
-                    return false;
-                }
-                if (getClass() != obj.getClass()) {
-                    return false;
-                }
-                ScriptUIEntry other = (ScriptUIEntry) obj;
-                if (scriptName == null) {
-                    if (other.scriptName != null) {
-                        return false;
-                    }
-                } else if (!scriptName.equals(other.scriptName)) {
-                    return false;
-                }
-                return true;
-            }
-
-            @Override
-            public int compareTo(ScriptUIEntry other) {
-                if (other == null) {
-                    return 1;
-                }
-                return scriptName.compareTo(other.scriptName);
-            }
-
-        }
-    }
+		HttpFuzzerMessageProcessorUIHandler<FuzzerHttpMessageScriptProcessorAdapter, FuzzerHttpMessageScriptProcessorAdapterUI> {
```

Comment:
Since 200+ lines are marked as changed here it's really hard to tell what's relevant.
Did you change tabs or line endings or something?
(The content on this line _appears_ to be the same as the first removed line.)

Formatting changes should be done in a separate PR. 
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting